stable/artifactory-ha/values.yaml

```diff
@@ -720,7 +720,11 @@ artifactory:
             {{- with .kmsCryptoMode }}
               <kmsCryptoMode>{{ . }}</kmsCryptoMode>
             {{- end }}
+            {{- if .useInstanceCredentials }}
```

Comment:
Wouldn't it be more elegant to have `useInstanceCredentials: false` as the default and avoid the `if` clause?